src/Tools/Vsdconfig/Vsdconfig.targets

```diff
@@ -5,7 +5,7 @@
           Inputs="@(VsdConfigXml);$(IntermediateOutputPath)\$(AssemblyName).dll"
           Outputs="$(OutDir)\$(AssemblyName).vsdconfig"
           Condition="'$(SkipGenerateVsdconfig)' != 'true'">
-    <Exec Command="&quot;$(VSSDK140Install)VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
+    <Exec Command="&quot;$(VS140COMNTOOLS)..\..\VSSDK\VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
```

Comment:
I did this work on a fresh machine which had only VS 2015 CTP6 installed and `VSSDK140Install` was not defined.  It is defined on my long running dev machine though.  After looking at the output of env this was the common variable between the two machines. 

I had @amcasey take a look at this and he was fine with this switch.